fk/secretsend.go

```diff
@@ -79,9 +79,10 @@ https://download.fluidkeys.com#` + recipientEmail + `
 	if filename != "" {
 		secret, err = getSecretFromFile(filename, nil)
 
-		out.Print("---\n")
+		printFileDivider(filename)
```

Comment:
I suspect this should be the *basename* e.g. `aws-cert.pem` rather than `~/.ssh/aws-cert.pem` ... since the information we're transmitting does not include the full path.